Kepler-Server/src/main/java/org/alexdev/kepler/game/games/gamehalls/utils/GamePokerCombination.java

```diff
@@ -0,0 +1,111 @@
+package org.alexdev.kepler.game.games.gamehalls.utils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public enum GamePokerCombination {
+    FIVE_OF_A_KIND("a Five of a kind"),
```

Comment:
I will change this to have proper texts using any tooling present in the code. This is not really great (TBH I thought that I would remove it before sending this PR / keep this for my code as original version doesn't perform this).